deepquest/__main__.py

```diff
@@ -33,6 +37,17 @@ def score(args):
     from . import score
     score(args.files)
 
+def set_gpu_id(gpuid):
+    import os
+    os.environ["CUDA_DEVICE_ORDER"] = "PCI_BUS_ID"
+    gpustr = ''
+    for g in gpuid:
+        gpustr += str(g) + ','
```

Comment:
It is more intuitive to write
`gpu-id: 1,2` 
than 
`gpu-id: 1 2`
in the config file, but since both could happen, I would consider both cases with something like:
```
gpuid = gpuid.split(',') if ',' in gpuid else gpuid.split()    
```
BTW, this also works if there is more than 10 GPUs (i.e. gpuid > 9, with `gpuid = '10'` resulting into two GPU IDs of 1 and 0 in the current version of the code.